### PR TITLE
Feast Helm charts and build script

### PR DESCRIPTION
**Pull request by davidheryanto**
This pull request provides updates to the Helm chart used to install Feast using Helm.
It also updates the Dockerfiles used to build Feast Core and Serving.
Currently it assumes the version to be `0.3.0-alpha.1`.

Steps to test this chart can be seen in `README.md` in `infra/charts/feast` folder.

```
helm repo add feast-charts https://feast-charts.storage.googleapis.com
helm repo update

RELEASE_NAME=test
helm install feast-charts/feast --name $RELEASE_NAME --version 0.3.0-alpha.1 \
 -f https://raw.githubusercontent.com/davidheryanto/feast/feast-infra-helm-charts/infra/charts/feast/values-demo.yaml
```

> Current build is based on this commit sha afc4308

### CODE REVIEW

File: infra/charts/feast/values-external-store.yaml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# TODO @dheryanto
+#
```
Comment (woop) [inline infra/charts/feast/values-external-store.yaml]: should we perhaps exclude these? Dont see value in adding this.

**Comment by woop** [comment]
> https://raw.githubusercontent.com/davidheryanto/feast/feast-infra-helm-charts/infra/charts/feast/values-demo.yaml

Is this available also from /gojek/feast? or only from your repo?

**Comment by davidheryanto** [comment]
> > https://raw.githubusercontent.com/davidheryanto/feast/feast-infra-helm-charts/infra/charts/feast/values-demo.yaml
> 
> Is this available also from /gojek/feast? or only from your repo?

Yes it will be available in this path: 
`FEAST_REPO/infra/charts/feast/value-demo.yaml` when it is merged to 0.3-dev

**Comment by davidheryanto** [comment]
/hold
Missing sample configuration for production

**Comment by davidheryanto** [comment]
/hold cancel
Current chart is tested and ok in dev environment

**Pull request merged**